napari/_vendor/qt_json_builder/qt_jsonschema_form/widgets.py

```diff
@@ -599,6 +600,20 @@ def configure(self):
         self.setGraphicsEffect(self.opacity)
         self.opacity.setOpacity(1)
 
+class DaskSettingsWidget(SchemaWidgetMixin, MagicDaskSettingsWidget):
+
+    @state_property
+    def state(self) -> dict:
+        return self._widget()
+
+    @state.setter
+    def state(self, state: dict):
+        self.setValue(state)
```

Comment:
These two may live in BaseMagicSetting As they will be shared by any magicgui based settings field?